Statistics.ipynb

```diff
@@ -171,14 +171,14 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Images with annotations 169\n",
+      "Images with annotations 166\n",
```

Comment:
e corretto che ci siano meno annotazioni di prima?

Reply:
Sì, abbiamo condotto un approfondito lavoro di revisione del dataset che ha portato anche alla cancellazione di alcune immagini o al cambio della classe associata. È possibile che alcune immagini fossero annotate con il materiale ma non mostrassero una discarica, quindi sono state scartate o spostate a negative. In entrambi i casi non rientrano più nelle immagini annotate fine grain.